### PR TITLE
Update To 2.2.1

### PR DESCRIPTION
**Pull request by fbordeu**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by fbordeu** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -108,28 +111,29 @@ outputs:
...
-      - libboost-headers
-      - pytest
-      - vtk
-      - tbb
-      - tbb-devel
-      - matplotlib
-      - coverage
-      - pywin32  # [win]
+      run:
+        - {{ compiler('cxx') }}
```
Comment (jaimergp) [inline recipe/meta.yaml:115]: This is weird as a runtime dependency

File: recipe/meta.yaml [hunk 2]
```diff
@@ -108,28 +111,29 @@ outputs:
...
+        - python {{ python }}
```
Comment (jaimergp) [inline recipe/meta.yaml:116]:
```suggestion
        - python
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -25,13 +25,16 @@ outputs:
...
+        - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - cython                                 # [build_platform != target_platform]
+        - ninja                                  # [build_platform != target_platform]
         - {{ compiler('cxx') }}
         - cython
       host:
         - python
         - numpy
+        - ninja
```
[the remark this answers is not in the document]
Comment (fbordeu) [reply recipe/meta.yaml:37]: I think you are right

File: recipe/meta.yaml (hunk 1 above)
Comment (fbordeu) [reply recipe/meta.yaml:115]: the package "muscat-devenv" has all the dependencies to work on the Muscat projet "as a developer". Historically to have all the dependencies of Muscat (without muscat) the anwser was "conda install Muscat; conda uninstall Muscat". The idea here is to install muscat-devenv from conda-forge and Muscat from sources (git clone).

Is this a good idea??

File: recipe/meta.yaml (hunk 2 above)
Comment (fbordeu) [reply recipe/meta.yaml:116]: ok

File: recipe/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipe/meta.yaml:113]: Can't say if good or not, but surely an unusual purpose for a conda package published in a community channel. Usually these things are more suitable for an internal channel (e.g. anaconda.org/muscat).

Comment (jaimergp) [reply recipe/meta.yaml:113]: e.g. this wouldn't have passed staged-recipes without a good explanation.

**Pull request merged**